src/mscorlib/src/System/Text/DecoderNLS.cs

```diff
@@ -32,15 +32,6 @@ internal class DecoderNLS : Decoder, ISerializable
 
 #region Serialization
 
-        // Constructor called by serialization. called during deserialization.
-        internal DecoderNLS(SerializationInfo info, StreamingContext context)
```

Comment:
Removing this will break deserialization of this type (even though I see its implementation is to throw a not supported exception)